docker-compose.yml

```diff
@@ -4,17 +4,19 @@ services:
     container_name: backend
     build:
       context: .
-      dockerfile: Dockerfile
+      dockerfile: Dockerfile   
```

Comment:
Remove this unneeded whitespace (unfortunately we don't have a linter for this .yml file 😢 )

Reply:
done